Classes/DDFileLogger.m

```diff
@@ -31,13 +31,15 @@
 // So we use primitive logging macros around NSLog.
 // We maintain the NS prefix on the macros to be explicit about the fact that we're using NSLog.
 
-#define LOG_LEVEL 2
+#ifndef DD_NSLOG_LEVEL
+    #define DD_NSLOG_LEVEL 2
+#endif
```

Comment:
LOG_LEVEL is very generic and it could be confused with the actual Cocoalumberjack log level. Here this define is specifically only used for NSLogs which is why I suggested renaming it to be clearer.